TheFoodieNetwork/app/src/main/java/com/fbu/thefoodienetwork/activities/SignUpActivity.java

```diff
@@ -0,0 +1,136 @@
+package com.fbu.thefoodienetwork.activities;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.text.TextUtils;
+import android.util.Log;
+import android.util.Patterns;
+import android.view.View;
+import android.widget.EditText;
+import android.widget.Toast;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+import com.fbu.thefoodienetwork.R;
+import com.fbu.thefoodienetwork.databinding.ActivitySignUpBinding;
+import com.parse.ParseException;
+import com.parse.ParseUser;
+import com.parse.SignUpCallback;
+
+public class SignUpActivity extends AppCompatActivity {
+    private static final String TAG = "SignUpActivity";
+    private static final int PASS_UNMATCH = -1;
+    private static final int PASS_INVALID_LENGTH = 1;
+    private static final int PASS_VALID = 0;
```

Comment:
Nice!!  Something you could use here, instead, would be an enum as it could provide you with a little more flexibility :-)